### DISCUSSION TITLE
Clone functione

### DISCUSSION THREAD

**Discussion by MiaCoco** [ChiLiubio/microeco#282] [Q&A] [answered]
Please could you help me with this error, maybe it is basic problem, but I tried everything so far. :/
group_CW <- clone(dataset)
Error in clone(dataset) : could not find function "clone"

**Comment by ChiLiubio** [comment, marked as answer]
Please load the microeco package with `library(microeco)`.
`clone` function is one function from microeco package.

**Reply by MiaCoco** [reply]
Thank you soo much, it was already loaded, but I don't know why it did not work. But now everything worked out  fine.
